### ISSUE TITLE
Hangman exercise says language-specific files will contain explanations; no such files exist

### ISSUE THREAD

**Issue by rmunn** [exercism/fsharp#549]
1. Run `exercism f fsharp` to fetch the hangman exercise (if you're at that point).
1. The README says "You'll need to install an FRP library for this, this will be described in the language/track specific files of the exercise."
1. Nothing in the F#-specific files suggests which FRP library should be downloaded, or even a mesasge like "You don't need an FRP library for this; here's a skeleton of an FRP approach that you can copy and flesh out."

**Comment by ErikSchierboom** [comment]
Yeah, I've been struggling a lot with the FRP approach, chiefly because of my lack of experience with such an approach. Do you perhaps have any experience with FRP? And if so, would you be willing to help us re-design this exercise to use proper FRP?

**Comment by rmunn** [comment]
I don't have any experience with FRP myself either, so I'm perhaps not the best person to work on this. A couple resources that might be useful: [a good intro to FRP concepts in Javascript](https://gist.github.com/staltz/868e7e9bc2a7b8c1f754), and [the Gjallarhorn library](https://reedcopsey.github.io/Gjallarhorn/) which is designed to take the Observable stuff from .Net and make it more idiomatic F#. IMHO, Gjallarhorn might be the best library to use for this exercise, as the Gjallarhorn documentation is actually accessible (as opposed to the Observable docs, which are about as dense as an overgrown jungle, and about as inaccessible as that jungle as well).
